src/test/validation_chainstatemanager_tests.cpp

```diff
@@ -15,15 +18,62 @@
 
 #include <boost/test/unit_test.hpp>
 
-BOOST_FIXTURE_TEST_SUITE(validation_chainstatemanager_tests, TestingSetup)
+struct ChainTestingSetup : public BasicTestingSetup {
+    boost::thread_group threadGroup;
+
+    explicit ChainTestingSetup(const std::string& chainName = CBaseChainParams::MAIN, const std::vector<const char*>& extra_args = {})
```

Comment:
You don't need these ctor arguments. Just remove them and call `BasicTestingSetup()` (which will use the default arguments for its constructor).